tests/ngff/test_ngff.py

```diff
@@ -392,6 +398,7 @@ def test_rename_channel(channels_and_random_5d, arr_name, new_channel):
         assert dataset.metadata.omero.channels[0].label == new_channel
 
 
+# @pytest.mark.skip(reason="broken")
```

Comment:
@aliddell This test does pass now. Can you document why it was marked 'broken'?

Reply:
I don't know if it's worth documenting why it was broken. It was the last test not involving tifffile that I needed to get to pass. I'll just remove the mark.